src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectParamsExtensions.cs

```diff
@@ -30,15 +30,11 @@ public static bool IsValid(this ConnectParams parameters, out string errorMessag
             }
             else if (string.IsNullOrEmpty(parameters.Connection.AuthenticationType) || parameters.Connection.AuthenticationType == "SqlLogin")
             {
-                // For SqlLogin, username/password cannot be empty
+                // For SqlLogin, username cannot be empty
```

Comment:
you should probably make sure password is not set to required in capabilities service (used by carbon)